content/cloud-docs/users-teams-organizations/permissions.mdx

```diff
@@ -27,10 +27,11 @@ There are also some actions within an organization that are only available to ow
 
 Permissions for the owners team include:
 
-- Manage workspaces (see [Organization Permissions][] below; equivalent to admin permissions on every workspace)
-- Manage policies (see [Organization Permissions][] below)
-- Manage policy overrides (see [Organization Permissions][] below)
-- Manage VCS settings (see [Organization Permissions][] below)
+- Manage workspaces (refer to [Organization Permissions][] below; equivalent to admin permissions on every workspace)
+- Manage policies (refer to [Organization Permissions][] below)
+- Manage policy overrides (refer to [Organization Permissions][] below)
+- Manage VCS settings (refer to [Organization Permissions][] below)
+- Manage Private Registry (refer to [Organization Permissions][] below)
 - Publish private modules (owners only)
```

Comment:
I think this is no longer `(owners only)`? Managing the private registry gives users permission to publish modules

Reply:
OH WHAT A GOOD CATCH!!!! let me fix this - thank you

Reply:
Okay I'm going to remove that line actually because it honestly is already included in our "manage private registry" statement.